xml/System.Text.Json.Serialization/JsonConverter.xml

```diff
@@ -37,9 +37,9 @@
         <Parameter Name="typeToConvert" Type="System.Type" />
       </Parameters>
       <Docs>
-        <param name="typeToConvert">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="typeToConvert">The type is checked as to whether it can be converted.</param>
```

Comment:
```suggestion
        <param name="typeToConvert">The type to compare against.</param>
```

Reply:
I don't necessarily agree with this change. What are we comparing against? I prefer the original.

```suggestion
        <param name="typeToConvert">The type of the object to check whether it can be converted by this converter instance.</param>
```